tools/quadrados-gen/src/main.cpp

```diff
@@ -31,7 +29,7 @@ struct Component
 /// Prints the help message of the program.
 static void printHelp()
 {
-    std::cerr << "Usage: quadrados generate <INPUT> <OUTPUT>" << std::endl;
+    std::cerr << "Usage: quadrados <INPUT> <OUTPUT>" << std::endl;
```

Comment:
```suggestion
    std::cerr << "Usage: quadrados-gen <INPUT> <OUTPUT>" << std::endl;
```

Reply:
The binaries should have different names, probably also requires a change in the cmake?